common/attest_plugin.h

```diff
@@ -20,6 +20,13 @@
 
 OE_EXTERNC_BEGIN
 
+// Note: V1 is OE_REPORT_HEADER_VERSION, for legacy report header
+// V2 is for legacy attestation header that, for SGX local and remote
+// attestation, requires a legacy report header to prefix the SGX report or
```

Comment:
can you mention the actual structure names in the comment?

Reply:
Sure.